src/routes.ts

```diff
@@ -576,6 +576,10 @@ export namespace responses {
 		return html` <page-error .message="${errorMessage}" .expand="${!notFullHeight}"></page-error>`;
 	}
 
+	export function renderEeOnly(): TemplateResult {
+		return html`Enterprise only`;
+	}
+
```

Comment:
This screen should never be visible, the UI for it gets hidden. I like the idea of an enterprise only page in case of a bug. https://linear.app/rivet-gg/issue/HUB-241/create-enterprise-only-page-instead-of-adding-empty-text